packages/pciutils/pciutils.spec

```diff
@@ -0,0 +1,51 @@
+%global debug_package %{nil}
+%global __strip %{_bindir}/true
```

Comment:
We shouldn't need these.

```suggestion
```

Reply:
Nice call! I added these when trying things but they are not needed!